packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1423,9 +1423,7 @@ abstract class FlutterCommand extends Command<void> {
             dartDefineConfigJsonMap[key] = value;
           });
         } on FormatException catch (err) {
-          throwToolExit('Json config define file "--${FlutterOptions
-              .kDartDefineFromFileOption}=$path" format err, '
-              'please fix first! format err:\n$err');
+          throwToolExit('Unable to parse $path. $err');
```

Comment:
any reason not to ask the user to fix the file? I'm thinking they may see this and think it's an internal error and file a bug.

Reply:
The details of the `FormatException` would hopefully lead users to figure out what needs to be fixed, but I think making a more explicit call to action is a good idea. Updated to this:

https://github.com/flutter/flutter/blob/59c709ff94e071df91b1d25ad63703f226045743/packages/flutter_tools/lib/src/runner/flutter_command.dart#L1426-L1429